Makefile

```diff
@@ -152,7 +152,8 @@ ${KOPS}:
 codegen:
 	go install k8s.io/kops/upup/tools/generators/...
 	${GOPATH_1ST}/bin/fitask --input-dirs k8s.io/kops/upup/pkg/fi/... \
-	    --go-header-file "hack/boilerplate/boilerplate.generatego.txt"
+	    --go-header-file "hack/boilerplate/boilerplate.generatego.txt" \
+	    --output-base ${GOPATH_1ST}/src/k8s.io/kops/
```

Comment:
Maybe quote this one too?

Reply:
Considering that the fitask path is unquoted too, a better solution was to just escape the spaces at source.
Also, the quoting on the header file path was not very useful.